batch/batch/server/server.py

```diff
@@ -110,14 +113,52 @@ def _next_task(self):
         self._task_idx += 1
         if self._task_idx < len(self._tasks):
             self._current_task = self._tasks[self._task_idx]
+        else:
+            self._delete_pvc()
 
     def _has_next_task(self):
         return self._task_idx < len(self._tasks)
 
+    def pvc(self):
+        if self._pvc is None:
+            self._pvc = v1.create_namespaced_persistent_volume_claim(
+                POD_NAMESPACE,
+                kube.client.V1PersistentVolumeClaim(
+                    metadata=kube.client.V1ObjectMeta(
+                        generate_name=f'job-{self.id}-',
+                        labels={'app': 'batch-job',
+                                'hail.is/batch-instance': instance_id}),
+                    spec=kube.client.V1PersistentVolumeClaimSpec(
+                        access_modes=['ReadWriteOnce'],
+                        volume_mode='Filesystem',
+                        resources=kube.client.V1ResourceRequirements(
+                            requests={'storage': POD_VOLUME_SIZE}),
+                        storage_class_name=STORAGE_CLASS_NAME)),
+                _request_timeout=KUBERNETES_TIMEOUT_IN_SECONDS)
+            log.info(f'created pvc name: {self._pvc.metadata.name} for job {self.id}')
+        return self._pvc
+
     def _create_pod(self):
         assert not self._pod_name
```

Comment:
Not for this PR, but did you do this to visually separate from `is not None` in the line below?

Reply:
Because otherwise, you typically use `is None`. I'd prefer a single None checking convention

Reply:
rectified